tests/components/netatmo/test_config_flow.py

```diff
@@ -0,0 +1,89 @@
+"""Test the Netatmo config flow."""
+from homeassistant import config_entries, data_entry_flow, setup
+from homeassistant.components.netatmo import config_flow
+from homeassistant.components.netatmo.const import (
+    DOMAIN,
+    OAUTH2_AUTHORIZE,
+    OAUTH2_TOKEN,
+)
+from homeassistant.helpers import config_entry_oauth2_flow
+
+from tests.common import MockConfigEntry
+
+CLIENT_ID = "1234"
+CLIENT_SECRET = "5678"
+
+
+async def test_abort_if_existing_entry(hass):
+    """Check flow abort when an entry already exist."""
+    MockConfigEntry(domain=DOMAIN).add_to_hass(hass)
+
+    flow = config_flow.NetatmoFlowHandler()
+    flow.hass = hass
+
+    result = await flow.async_step_user()
```

Comment:
We prefer to use the core interfaces as it will make the tests more robust.

```py
result = await hass.config_entries.flow.async_init(
    "netatmo", context={"source": config_entries.SOURCE_USER}
)
```